### ISSUE TITLE
Project seems like to be dead

### ISSUE THREAD

**Issue by Pedro1234-code** [coofcookie/Windows11Upgrade#33]
This project seems like to be dead - no updates, nothing

Is this really dead?

**Comment by sepodele** [comment]
Might be dead but I can confirm that it still works. :)

**Comment by Dark-Yox** [comment]
Why updating something if it's still working as intended, if you have any suggestions just write them instead of opening this kind of useless "issues"

**Comment by Aritz331** [comment]
@Dark-Yox yeah, it doesnt need any updates to keep working as it is; but I would suggest a design improvement

**Comment by Zhurama** [comment]
I sure hope you will release another update with a bit better functionality. I managed one DL of the ISO and it definitely works, but needs improvement. Please do consider updating.. The current version just hangs doing nothing after language selection. It's like it can't find the DL server. A lot of folks are going to need this less than 3 years from now. It's an excellent Idea, and has already saved a couple of PCs.. tyvvm.

**Comment by Pedro1234-code** [comment]
Because it is probaly not working to upgrade Windows 11 21H2 to 22H2, as now the installer checks if appraiseres.dll is present before starting the upgrade process

**Comment by justperry** [comment]
Doesn't work (anymore) here as well.
Agreed with another poster above to not update if it works but, at least for me, it doesn't.
